src/utils/hibp.js

```diff
@@ -0,0 +1,205 @@
+import got from 'got'
+import mozlog from './log.js'
+import AppConstants from '../app-constants.js'
+import { fluentError } from './fluent.js'
+const { HIBP_THROTTLE_MAX_TRIES, HIBP_THROTTLE_DELAY, HIBP_API_ROOT, HIBP_KANON_API_ROOT, HIBP_KANON_API_TOKEN } = AppConstants
```

Comment:
Does this use the latest HIBP API (v3 I think)?  If not, should we add it to 2023 tech-debt?

Reply:
There are two APIs of HIBP we use, the HIBP public API (v2, v3) is only used for the "latest breach" page. Depends on the premium design, we may or may not need that page anymore